components/tikv_alloc/src/jemalloc.rs

```diff
@@ -342,6 +342,7 @@ mod profiling {
     const OPT_PROF: &[u8] = b"opt.prof\0";
     const ARENAS_CREATE: &[u8] = b"arenas.create\0";
     const THREAD_ARENA: &[u8] = b"thread.arena\0";
+    const BACKGROUND_THREAD: &[u8] = b"background_thread\0";
```

Comment:
NIT: maybe we can consider using [`CStr`](https://doc.rust-lang.org/edition-guide/rust-2021/c-string-literals.html) type to avoid explicitly handle the cstring '\0'. But this approach may lead to some api changes.

Reply:
Try
```suggestion
    const BACKGROUND_THREAD: &[u8] = c"background_thread";
```

Reply:
Tried, but it still needs to pass the &[u8] to jemalloc-ctl which makes it even more complicated